app/scripts/fuzzydate.js

```diff
@@ -0,0 +1,78 @@
+const Timeunit = {
+  Year: "year",
+  Month: "month",
+  Day: "day",
+  Hour: "hour",
+  Minute: "minute",
+  Second: "second",
+}
+
+
+function absoluteFormat(unit, thisYear, time=false) {
+  let format = {}
+  switch (unit) { // fall through
+    case Timeunit.Second: // treated like minute, GH never shows seconds
+    case Timeunit.Minute:
+      if (time) {
+        format.minute = "2-digit"
+      }
+    case Timeunit.Hour:
+      if (time) {
+        format.hour = "2-digit"
+        format.timeZoneName = "short"
+      }
+    case Timeunit.Day:
+      format.day = "numeric"
+    case Timeunit.Month:
+      format.month = "short"
```

Comment:
I think the longer month format would look better. WDYT?

Reply:
GitHub itself uses the short month format. We'd run into overflow issues with the long format. I think the short format is fine.
<img width="941" alt="Bildschirmfoto 2021-05-19 um 09 06 49" src="https://user-images.githubusercontent.com/122697/118770479-aa286c80-b881-11eb-9223-fab077c465db.png">